README.md

```diff
@@ -99,20 +99,57 @@ Each concept exercise and concept have an `introduction.md` file. If you want th
 
 Concept placeholders must use the following format:
 
-```
+```markdown
 %{concept:<slug>}
 ```
 
 For example, if the track has a concept named `floating-point-numbers` then an `introduction.md.tpl` file can contain:
 
-```
+```markdown
 %{concept:floating-point-numbers}
 ```
 
 You can run `configlet generate` to generate the exercise's `introduction.md` for any exercise that has an `introduction.md.tpl` file. The generated `introduction.md` is identical to the `introduction.md.tpl`, except that concept placeholders are replaced with the contents of the concept's `introduction.md` file (minus its top-level heading). In the future, `configlet generate` will also increment the level of other headings by 1 (e.g. from `## My Heading` to `### My Heading`), but this is not yet implemented.
 
 ## Use in your track
 
+There are two ways that configlet can be kept up to date and used in a language track:
+
+- Using a GitHub workflow action
+- Calling directly
+
+### Action
+
+The action downloads configlet and adds it to the path where it can then be used, for example as part of CI.
+For further information, see [action.yml](./action.yml).
+
+#### Example
+
+The following would be in a `.yml` file in `./.github/workflows/`
+
+```yml
+name: Configlet CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  configlet:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Fetch configlet
+      uses: exercism/github-actions/configlet-ci@main
+
+    - name: Configlet linter
+      uses: configlet lint
```

Comment:
This workflow is untested.

```suggestion
name: Configlet

permissions:
  contents: read

on:
  workflow_dispatch:
  push:
    branches:
      - main
  pull_request:

jobs:
  configlet:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2

    - name: Fetch configlet
      uses: exercism/github-actions/configlet-ci@main

    - name: Configlet linter
      uses: configlet lint
```

@SleeplessByte will yell at me for suggesting to only run `push` builds on `main` but for configlet I think it's sufficient. Having `workflow_dispatch` as manual trigger is always useful. For those constructed examples of edge cases where one actually needs `push` in a PR branch (that imo will never occur for configlet), `workflow_dispatch` can be used. 

[Specifying the permissions will set all other permissions to `none`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions). Therefore it doesn't really matter if we pin the actions or not. If they're compromised, the worst that can happen would be abusing the runner resources. If that happens to `actions/checkout`, that's GH's fault.

Reply:
I won't yell but yes, why would we do this 👁️

Reply:
Because the config needs to be valid after being merged, the state of the config on the branch on its own is largely irrelevant. It's wasteful to have builds that are useless 99.9999% of the time they run.